src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const lines = sourceString.split(';');
+
+  lines.forEach((line) => {
+    const [property, value] = line.split(':');
+
+    if (property && value) {
+      const trimmedProperty = property.trim();
+      const trimmedValue = value.trim();
+
+      styles[trimmedProperty] = trimmedValue;
```

Comment:
Checking for the truthiness of `property` and `value` is a good practice, but it may lead to unintended behavior if property or value are strings that are considered falsy (e.g., an empty string). It might be better to check that the strings are not empty after trimming.